pkg/api/v1/defaults.go

```diff
@@ -101,6 +102,9 @@ func SetDefaults_Service(obj *v1.Service) {
 	if obj.Spec.SessionAffinity == "" {
 		obj.Spec.SessionAffinity = v1.ServiceAffinityNone
 	}
+	if obj.Spec.SessionAffinity == v1.ServiceAffinityClientIP && obj.Spec.ClientIPAffinityConfig == nil {
+		defaultServiceAffinityConfig(obj.Spec.ClientIPAffinityConfig)
```

Comment:
I am so sorry about that I didn't test it. It's fixed now and I created a UT to cover this, PTAL. Thanks!